Referenced by issue #80694, bpy.ops.script.reload() seg fault
